website/docs/components/tabs/partials/specifications/design-guidelines.md

```diff
@@ -113,46 +73,35 @@ Refer to [HashiCorp's Style, Language, and Voice Guidelines](https://docs.google
 
 ---
 
-## Accessibility
-
-### Animation
-
-No indicator animation will be present for users that have enabled `prefers-reduced-motion`.
-
 ### Keyboard navigation
 
 Focus
 
-<section>
-  <Hds::Badge @color="neutral" @text="Tab" @size="small" />
-</section>
+<Hds::Badge @color="neutral" @text="Tab" @size="small" />
 
 ![Keyboard tab navigation example](/assets/components/tabs/tabs-accessibility-tab-01.png)
 
 Activate tab to display matching content area
 
-<section>
-  <Hds::Badge @color="neutral" @text="Spacebar" @size="small" />
-  <Hds::Badge @color="neutral" @text="Enter" @size="small" />
-</section>
+<Hds::Badge @color="neutral" @text="Spacebar" @size="small" />
+<Hds::Badge @color="neutral" @text="Enter" @size="small" />
 
 ![Keyboard tab navigation example](/assets/components/tabs/tabs-accessibility-spacebar_enter.png)
 
 Move between tabs
 
-<section>
-  <Hds::Badge @color="neutral" @text="Arrow left" @isIconOnly={{true}} @icon="arrow-left" @size="small" />
-  <Hds::Badge @color="neutral" @text="Arrow right" @isIconOnly={{true}} @icon="arrow-right" @size="small" />
-</section>
+<Hds::Badge @color="neutral" @text="Arrow left" @isIconOnly={{true}} @icon="arrow-left" @size="small" />
+<Hds::Badge @color="neutral" @text="Arrow right" @isIconOnly={{true}} @icon="arrow-right" @size="small" />
 
 ![Keyboard tab navigation example](/assets/components/tabs/tabs-accessibility-arrow_keys.png)
 
 Move to interactive element within the content area
 
-<section>
-  <Hds::Badge @color="neutral" @text="Tab" @size="small" />
-</section>
+<Hds::Badge @color="neutral" @text="Tab" @size="small" />
 
 ![Keyboard tab navigation example](/assets/components/tabs/tabs-accessibility-tab-02.png)
 
-_Banner (informational):_ If focus is on the first tab, `left arrow` moves focus to the last tab. If focus is on the last tab, `right arrow` moves focus to the first tab.
+!!! Info
+
+ If focus is on the first tab, `left arrow` moves focus to the last tab. If focus is on the last tab, `right arrow` moves focus to the first tab.
+!!!
```

Comment:
Should I delete some of the above content and/or move to "Guidelines" or somewhere?

Reply:
[Per the wiki](https://github.com/hashicorp/design-system/blob/main/wiki/Website-Editing-existing-content.md#steps-to-getting-started), I think this gets renamed and moved to a different section entirely: `Rename design-guidelines.md to guidelines.md and move it to /guidelines`.

Reply:
Up to you which order but parts of this can stay in an `anatomy.md` file and the guidelines move as @MelSumner indicated

Reply:
Thanks I'll make those changes.